packages/hydra-typegen/src/commands/typegen/index.ts

```diff
@@ -136,34 +136,74 @@ types don't much the metadata definiton`,
       strict: flags.strict,
     } as IConfig
   }
-
-  async buildGeneratorConfigs(config: IConfig): Promise<GeneratorConfig[]> {
+  async buildGeneratorConfigs(config: IConfig): Promise<{
```

Comment:
Just need an extra line here to satisfy linter ;)

https://github.com/Joystream/hydra/pull/531/checks#step:4:365 

```suggestion

  async buildGeneratorConfigs(config: IConfig): Promise<{
```